adoc/CaaSP_DH2X_Install_Guide.adoc

```diff
@@ -514,42 +550,49 @@ Execute the SAP Data Hub upgrade as described in the official instructions. You
+=== Installing a Secure Private Docker Registry
 
-** link:http://port.us.org/[Portus]
+To meet the Data Hub requirements you also need a Docker Registry.
+The Portus project provides an easy way to build and manage an own Docker Registry.
+For more information, see the project homepage at http://port.us.org/.
 
-First you need to create a dedicated server for your Docker registry and Portus stack.
+To set up Portus in a dedicated virtual machine, do:
 
+. Create a suitable a virtual machine:
++
 ----
-# sudo virt-install --name portus-dr --ram 8192 --disk path=/var/lib/libvirt/VMS/portus-dr.qcow2,size=40 --vcpus 4 --os-type linux --os-variant generic --network bridge=common --graphics none --console pty,target_type=serial --location '/var/lib/libvirt/isos/SLE-12-SP4-Server-DVD-x86_64-GM-DVD1.iso' --extra-args 'console=ttyS0,115200n8 serial ifcfg=eth0=10.10.10.11/24,10.10.10.1,10.10.10.11,suse-sap.net hostname=portus-dr domain=suse-sap.net Textmode=1'
+# sudo virt-install --name portus-dr --ram 8192 \
+  --disk path=/var/lib/libvirt/VMS/portus-dr.qcow2,size=40 --vcpus 4 \
+  --os-type linux --os-variant generic --network bridge=common --graphics none \
+  --console pty,target_type=serial \
+  --location '/var/lib/libvirt/isos/SLE-12-SP4-Server-DVD-x86_64-GM-DVD1.iso' \
+  --extra-args 'console=ttyS0,115200n8 serial ifcfg=eth0=10.10.10.11/24,10.10.10.1,10.10.10.11,suse-sap.net hostname=portus-dr domain=suse-sap.net Textmode=1'
 ----
 
-In our example this server will be connected to another local bridge which provides common services (DNS, SMT, Docker-registry) for the Data Hub stack.
-
-Our Portus deployment is based on container, and orchestrated locally with *docker-compose*.
-
-Portus *docker-compose* deployment requires an up-to-date release of *docker-compose*.
-
+. In this example, the Portus server will be connected to a local bridge providing common services (DNS, SMT, and Docker Registry) for the SAP Data Hub 2 stack.
+Our Portus deployment is container-based and orchestrated locally with `docker-compose`.
+Portus `docker-compose` deployment requires an up-to-date release of `docker-compose`.
+To install `docker-compose`, use:
++
 ----
 sudo curl -L "https://github.com/docker/compose/releases/download/1.24.1/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
 sudo chmod +x /usr/local/bin/docker-compose
 sudo ln -s /usr/local/bin/docker-compose /usr/bin/docker-compose
 ----
 
-Now you can simply clone the Portus repository. Adapt the *.env* and the *nginx* configuration to your naming convention.
-
+. Clone the Portus Git repository:
++
 ----
 # git clone https://github.com/SUSE/Portus.git /tmp/Portus-DR
 # mv /tmp/Portus-DR/examples/compose ./portus
 # cd portus
 ----
 
-Edit both *.env* and *nginx/nginx.conf*.
-As an example, this is how our configuration looks like :
+// ^ FIXME: Why first clone to /tmp and the move that cloned repo $somewhere_else instead of cloning to $somewhere_else in the first place?
```

Comment:
Maybe a fixme for Ulrich

Reply:
This part was written by Christophe (cledorze@suse.com).
We should as him for comments on this section.

Reply:
@cledorze Is there a reason for checking the Portus repo out in this way?

Reply:
The thing is that we're altering files in a repo later in the doc.
But you're right you should delete it.
Also I don't think that you should replace the main repo docker-compose.yml by the one coming from https://gist.githubusercontent.com/Patazerty/d05652294d5874eddf192c9b633751ee/raw/6bf4ac6ba14192a1fe5c337494ab213200dd076e/docker-compose.yml which points to a non-supported S.I git repo.

Let me ask him if we could commit it to the main Portus repo

Reply:
Thanks! What I had completely missed priginally is that you only moved one of the subdirectories out of the repo. That makes sense to me (as long as updating via Git is not a use case which it does not appear to be). So, I left this part in and just removed my FIXME comment.

As to the Gist download -- yes, that looked a bit out of place to me too in "enterprise" documentation, because it seems like a rather unofficial source. Didn't seem urgent to me though. :)